README.md

```diff
@@ -8,13 +8,22 @@
     1. windows - Download installer from https://www.python.org/downloads/release/python-370/.
 1. Activate the virutal environment: `source env/bin/activate`
 1. Install all the dependencies in `requirements.txt` file: `pip install -r requirements.txt`
+1. Make a copy of `.env.sample` to `.env` and change the values of variables with original values. [Click here ](#example-env-file) for sample .env file.
```

Comment:
We need to add the `env.sample` file too.

Reply:
Added Roger Copy :joy: